moveit_ros/planning/planning_request_adapter_plugins/CMakeLists.txt

```diff
@@ -20,7 +20,6 @@ ament_target_dependencies(${MOVEIT_LIB_NAME}
     rclcpp
     pluginlib
     moveit_core
-    moveit_planning_scene
```

Comment:
don't see a target_link_libraries for this - was it not needed?

Reply:
`moveit_planning_scene` is a library included in the package `moveit_core` which is already included.